openedx/core/djangoapps/user_authn/views/login.py

```diff
@@ -476,8 +485,10 @@ def _inner(request):  # pylint: disable=missing-docstring
         # the enrollment API, we want to prevent the student views from
         # updating enrollments.
         if "enrollment_action" in modified_request:
+            set_custom_metric('shim_del_enrollment_action', modified_request["enrollment_action"])
```

Comment:
Minor minor nit: You could switch these to using `get` like the other ones if you want to be paranoid about it.